config/knooppunt.yml

```diff
@@ -22,4 +22,4 @@ mcsdadmin:
 # Nuts node configuration  
 nuts:
   # Whether to enable the Nuts node component
-  enabled: true
+  enabled: false
```

Comment:
I would just change the default in `DefaultConfig()`, and keep this on `true`, for easier testing

Reply:
Makes sense 🫣